file_handler.cpp

```diff
@@ -1,124 +1,115 @@
 // Based Off http://www.boost.org/doc/libs/1_55_0/doc/html/boost_asio/example/cpp11/http/server/request_handler.cpp
-#include "base_handler.hpp"
+#include "file_handler.hpp"
 #include <fstream>
 #include <sstream>
 
 namespace http {
 namespace server {
-	class file_handler: public base_handler {
-		public:
-		explicit file_handler(const std::string& doc){
-			docroot = doc;
+
```

Comment:
This is really nice. It was great that you were able to modularize classes and definitions and create a header file for the file_handler class. These little changes really help with the overall organization of our code.